tasties_app/tests/conftest.py

```diff
@@ -1,7 +1,30 @@
 import pytest
 from django.contrib.auth.models import User
 from django.utils import timezone
-from tasties_app.models import Category, Ingredient, Rating, Recipe
+from tasties_app.models import Category, Ingredient, Rating, Recipe, Comment
+from django.core.validators import ValidationError
+
+
+@pytest.fixture
+def comment():
+    """
+    in this generate function we create User, Category and Recipe
+    objects to fill Comment class and used to test
+     """
+    try:
+        user = User.objects.create_user(username='test_user', password='password')
+        recipe = Recipe(title='Test Recipe1', author_id=user, description='Test Description1',
+                        directions='Test Directions1', publication_date=timezone.now(), minutes_to_make=1,
+                        recipe_picture='recipe.jpeg')
+        recipe.clean_fields()
+        recipe.save()
+        comment = Comment(author_id=user, recipe_id=recipe, publication_date=timezone.now(),
+                          comment_text='test comment')
+        comment.clean_fields()
+        comment.save()
+    except ValidationError as e:
+        raise (e)
```

Comment:
Same as I asked before, why do you need to catch this error?

Reply:
if you attention Im used `clean_fields()` to check if all fields legal ...

Reply:
if something wrong throw to me type the error.

Reply:
Doesn't `clean_fields()` raise a validation error if something is wrong? why do you need to raise it?

Reply:
i deleted.